src/dom/index.js

```diff
@@ -83,10 +83,21 @@ export function setAccessor(node, name, old, value, isSvg) {
 		}
 		if (value && typeof value==='object') {
 			if (typeof old!=='string') {
-				for (let i in old) if (!(i in value)) node.style[i] = '';
+				for (let i in old) if (!(i in value)) {
+					if (~i.indexOf('-')) {
```

Comment:
We should check what the size is when doing `if (i[0]=='-')` - we only care about using setProperty for custom properties, so it'll always be a leading `--`.